<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Intro RNA-seq LCG-UNAM 2026</title>
  <meta name="description" content="Intro RNA-seq LCG-UNAM 2026" />
  <meta name="generator" content="bookdown 0.46 and GitBook 2.6.7" />

  <meta property="og:title" content="Intro RNA-seq LCG-UNAM 2026" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="lcolladotor/rnaseq_LCG-UNAM_2026" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Intro RNA-seq LCG-UNAM 2026" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="icon_32.png" type="image/x-icon" />

<link rel="next" href="introducci√≥n-a-r-y-positron.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link href="libs/htmltools-fill-0.5.9/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.12.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.2/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.2/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.25.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.25.2/plotly-latest.min.js"></script>


<style type="text/css">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#code-of-conduct"><i class="fa fa-check"></i>Code of Conduct</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#course-schedule"><i class="fa fa-check"></i>Course Schedule</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#external-links"><i class="fa fa-check"></i>External links</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#course-prerequisites"><i class="fa fa-check"></i>Course Prerequisites</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#getting-set-up"><i class="fa fa-check"></i>Getting set up</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-download-r-for-windows"><i class="fa fa-check"></i>How to Download R for Windows</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-download-r-for-the-mac"><i class="fa fa-check"></i>How to Download R for the Mac</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-download-rstudio-ide"><i class="fa fa-check"></i>How to Download <code>RStudio IDE</code></a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-download-positron"><i class="fa fa-check"></i>How to Download Positron</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-download-git"><i class="fa fa-check"></i>How to Download Git</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#create-your-github-account"><i class="fa fa-check"></i>Create your GitHub account</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-apply-for-github-education-and-configure-github-copilot"><i class="fa fa-check"></i>How to Apply for GitHub Education and configure GitHub Copilot</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-get-air-auto-code-formatter"><i class="fa fa-check"></i>How to get Air (auto-code formatter)</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#course-evaluation"><i class="fa fa-check"></i>Course Evaluation</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#r-session-information"><i class="fa fa-check"></i>R session information</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#license"><i class="fa fa-check"></i>License</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html"><i class="fa fa-check"></i><b>1</b> Introducci√≥n a R y Positron</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#r"><i class="fa fa-check"></i><b>1.1</b> R</a></li>
<li class="chapter" data-level="1.2" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#github"><i class="fa fa-check"></i><b>1.2</b> GitHub</a></li>
<li class="chapter" data-level="1.3" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#github-education-y-el-uso-de-ia-para-escribir-c%C3%B3digo"><i class="fa fa-check"></i><b>1.3</b> GitHub Education y el uso de IA para escribir c√≥digo</a></li>
<li class="chapter" data-level="1.4" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#positron"><i class="fa fa-check"></i><b>1.4</b> Positron</a></li>
<li class="chapter" data-level="1.5" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#rstudio"><i class="fa fa-check"></i><b>1.5</b> RStudio</a></li>
<li class="chapter" data-level="1.6" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#repositorio-de-notas-para-el-curso"><i class="fa fa-check"></i><b>1.6</b> Repositorio de notas para el curso</a></li>
<li class="chapter" data-level="1.7" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#material-del-curso"><i class="fa fa-check"></i><b>1.7</b> Material del curso</a></li>
<li class="chapter" data-level="1.8" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#actividad-postcards"><i class="fa fa-check"></i><b>1.8</b> Actividad postcards</a>
<ul>
<li class="chapter" data-level="1.8.1" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#create-beautiful-simple-personal-websites"><i class="fa fa-check"></i><b>1.8.1</b> Create Beautiful, Simple Personal Websites</a></li>
<li class="chapter" data-level="1.8.2" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#instalaci%C3%B3n"><i class="fa fa-check"></i><b>1.8.2</b> Instalaci√≥n</a></li>
<li class="chapter" data-level="1.8.3" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#plantillas"><i class="fa fa-check"></i><b>1.8.3</b> Plantillas</a></li>
<li class="chapter" data-level="1.8.4" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#crear-un-nuevo-proyecto-en-rstudio-selecci%C3%B3n-interactiva"><i class="fa fa-check"></i><b>1.8.4</b> Crear un nuevo proyecto en RStudio (selecci√≥n interactiva)</a></li>
<li class="chapter" data-level="1.8.5" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#crear-un-nuevo-proyecto"><i class="fa fa-check"></i><b>1.8.5</b> Crear un nuevo proyecto</a></li>
<li class="chapter" data-level="1.8.6" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#configurar-git-y-github"><i class="fa fa-check"></i><b>1.8.6</b> Configurar Git y GitHub</a></li>
<li class="chapter" data-level="1.8.7" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#elegir-una-plantilla"><i class="fa fa-check"></i><b>1.8.7</b> Elegir una plantilla</a></li>
<li class="chapter" data-level="1.8.8" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#editar-con-su-informaci%C3%B3n"><i class="fa fa-check"></i><b>1.8.8</b> Editar con su informaci√≥n</a></li>
<li class="chapter" data-level="1.8.9" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#desplegar-la-p%C3%A1gina"><i class="fa fa-check"></i><b>1.8.9</b> Desplegar la p√°gina</a></li>
<li class="chapter" data-level="1.8.10" data-path="introducci√≥n-a-r-y-positron.html"><a href="introducci√≥n-a-r-y-positron.html#compartan-su-p%C3%A1gina"><i class="fa fa-check"></i><b>1.8.10</b> Compartan su p√°gina</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introducci√≥n-a-bioconductor.html"><a href="introducci√≥n-a-bioconductor.html"><i class="fa fa-check"></i><b>2</b> Introducci√≥n a Bioconductor</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introducci√≥n-a-bioconductor.html"><a href="introducci√≥n-a-bioconductor.html#encontrando-paquetes-de-bioconductor"><i class="fa fa-check"></i><b>2.1</b> Encontrando paquetes de Bioconductor</a></li>
<li class="chapter" data-level="2.2" data-path="introducci√≥n-a-bioconductor.html"><a href="introducci√≥n-a-bioconductor.html#estructura-de-un-paquete-de-bioc"><i class="fa fa-check"></i><b>2.2</b> Estructura de un paquete de BioC</a></li>
<li class="chapter" data-level="2.3" data-path="introducci√≥n-a-bioconductor.html"><a href="introducci√≥n-a-bioconductor.html#las-dos-ramas-de-bioconductor-release-y-devel"><i class="fa fa-check"></i><b>2.3</b> Las dos ramas de Bioconductor: release y devel</a></li>
<li class="chapter" data-level="2.4" data-path="introducci√≥n-a-bioconductor.html"><a href="introducci√≥n-a-bioconductor.html#cursos-y-eventos"><i class="fa fa-check"></i><b>2.4</b> Cursos y eventos</a></li>
<li class="chapter" data-level="2.5" data-path="introducci√≥n-a-bioconductor.html"><a href="introducci√≥n-a-bioconductor.html#comunidad"><i class="fa fa-check"></i><b>2.5</b> Comunidad</a></li>
<li class="chapter" data-level="2.6" data-path="introducci√≥n-a-bioconductor.html"><a href="introducci√≥n-a-bioconductor.html#ejercicio-grupal"><i class="fa fa-check"></i><b>2.6</b> Ejercicio grupal</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="objetos-de-bioconductor-para-datos-de-expresi√≥n.html"><a href="objetos-de-bioconductor-para-datos-de-expresi√≥n.html"><i class="fa fa-check"></i><b>3</b> Objetos de Bioconductor para datos de expresi√≥n</a>
<ul>
<li class="chapter" data-level="3.1" data-path="objetos-de-bioconductor-para-datos-de-expresi√≥n.html"><a href="objetos-de-bioconductor-para-datos-de-expresi√≥n.html#summarizedexperiment"><i class="fa fa-check"></i><b>3.1</b> SummarizedExperiment</a></li>
<li class="chapter" data-level="3.2" data-path="objetos-de-bioconductor-para-datos-de-expresi√≥n.html"><a href="objetos-de-bioconductor-para-datos-de-expresi√≥n.html#ejercicio"><i class="fa fa-check"></i><b>3.2</b> Ejercicio</a></li>
<li class="chapter" data-level="3.3" data-path="objetos-de-bioconductor-para-datos-de-expresi√≥n.html"><a href="objetos-de-bioconductor-para-datos-de-expresi√≥n.html#isee"><i class="fa fa-check"></i><b>3.3</b> iSEE</a></li>
<li class="chapter" data-level="3.4" data-path="objetos-de-bioconductor-para-datos-de-expresi√≥n.html"><a href="objetos-de-bioconductor-para-datos-de-expresi√≥n.html#ejercicio-con-spatiallibd"><i class="fa fa-check"></i><b>3.4</b> Ejercicio con spatialLIBD</a></li>
<li class="chapter" data-level="3.5" data-path="objetos-de-bioconductor-para-datos-de-expresi√≥n.html"><a href="objetos-de-bioconductor-para-datos-de-expresi√≥n.html#ejemplo-con-hb_vta_projector_profiling"><i class="fa fa-check"></i><b>3.5</b> Ejemplo con Hb_VTA_projector_profiling</a></li>
<li class="chapter" data-level="3.6" data-path="objetos-de-bioconductor-para-datos-de-expresi√≥n.html"><a href="objetos-de-bioconductor-para-datos-de-expresi√≥n.html#comunidad-1"><i class="fa fa-check"></i><b>3.6</b> Comunidad</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="datos-de-rna-seq-a-trav√©s-de-recount3.html"><a href="datos-de-rna-seq-a-trav√©s-de-recount3.html"><i class="fa fa-check"></i><b>4</b> Datos de RNA-seq a trav√©s de recount3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="datos-de-rna-seq-a-trav√©s-de-recount3.html"><a href="datos-de-rna-seq-a-trav√©s-de-recount3.html#procesar-datos-crudos-fastq-con-speaqeasy"><i class="fa fa-check"></i><b>4.1</b> Procesar datos crudos (FASTQ) con SPEAQeasy</a></li>
<li class="chapter" data-level="4.2" data-path="datos-de-rna-seq-a-trav√©s-de-recount3.html"><a href="datos-de-rna-seq-a-trav√©s-de-recount3.html#proyectos-recount"><i class="fa fa-check"></i><b>4.2</b> Proyectos recount</a></li>
<li class="chapter" data-level="4.3" data-path="datos-de-rna-seq-a-trav√©s-de-recount3.html"><a href="datos-de-rna-seq-a-trav√©s-de-recount3.html#usar-recount3"><i class="fa fa-check"></i><b>4.3</b> Usar recount3</a></li>
<li class="chapter" data-level="4.4" data-path="datos-de-rna-seq-a-trav√©s-de-recount3.html"><a href="datos-de-rna-seq-a-trav√©s-de-recount3.html#ejercicio-1"><i class="fa fa-check"></i><b>4.4</b> Ejercicio</a></li>
<li class="chapter" data-level="4.5" data-path="datos-de-rna-seq-a-trav√©s-de-recount3.html"><a href="datos-de-rna-seq-a-trav√©s-de-recount3.html#comunidad-2"><i class="fa fa-check"></i><b>4.5</b> Comunidad</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html"><i class="fa fa-check"></i><b>5</b> Modelos estad√≠sticos</a>
<ul>
<li class="chapter" data-level="5.1" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#exploremodelmatrix"><i class="fa fa-check"></i><b>5.1</b> ExploreModelMatrix</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#ejemplo-1"><i class="fa fa-check"></i><b>5.1.1</b> Ejemplo 1</a></li>
<li class="chapter" data-level="5.1.2" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#ejemplo-2"><i class="fa fa-check"></i><b>5.1.2</b> Ejemplo 2</a></li>
<li class="chapter" data-level="5.1.3" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#ejemplo-3"><i class="fa fa-check"></i><b>5.1.3</b> Ejemplo 3</a></li>
<li class="chapter" data-level="5.1.4" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#ejercicio-2"><i class="fa fa-check"></i><b>5.1.4</b> Ejercicio</a></li>
<li class="chapter" data-level="5.1.5" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#para-aprender-m%C3%A1s"><i class="fa fa-check"></i><b>5.1.5</b> Para aprender m√°s</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#datos-de-srp045638"><i class="fa fa-check"></i><b>5.2</b> Datos de SRP045638</a></li>
<li class="chapter" data-level="5.3" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#normalizaci%C3%B3n-de-datos"><i class="fa fa-check"></i><b>5.3</b> Normalizaci√≥n de datos</a></li>
<li class="chapter" data-level="5.4" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#expresi%C3%B3n-diferencial"><i class="fa fa-check"></i><b>5.4</b> Expresi√≥n diferencial</a></li>
<li class="chapter" data-level="5.5" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#visualizando-genes-de"><i class="fa fa-check"></i><b>5.5</b> Visualizando genes DE</a></li>
<li class="chapter" data-level="5.6" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#ejercicio-3"><i class="fa fa-check"></i><b>5.6</b> Ejercicio</a></li>
<li class="chapter" data-level="5.7" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#comunidad-3"><i class="fa fa-check"></i><b>5.7</b> Comunidad</a></li>
<li class="chapter" data-level="5.8" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#ejercicio-respuesta"><i class="fa fa-check"></i><b>5.8</b> Ejercicio: respuesta</a></li>
<li class="chapter" data-level="5.9" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#sobre-centering-y-scaling-en-heatmaps"><i class="fa fa-check"></i><b>5.9</b> Sobre <em>centering</em> y <em>scaling</em> en heatmaps</a></li>
<li class="chapter" data-level="5.10" data-path="modelos-estad√≠sticos.html"><a href="modelos-estad√≠sticos.html#espec%C3%ADficaciones-del-proyecto"><i class="fa fa-check"></i><b>5.10</b> Espec√≠ficaciones del proyecto</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="pl√°tica-de-maria-guti√©rrez-arcelus.html"><a href="pl√°tica-de-maria-guti√©rrez-arcelus.html"><i class="fa fa-check"></i><b>6</b> Pl√°tica de Maria Guti√©rrez-Arcelus</a></li>
<li class="chapter" data-level="7" data-path="rna-seq-en-el-mundo-real-y-casi-todo-lo-que-puede-salir-mal.html"><a href="rna-seq-en-el-mundo-real-y-casi-todo-lo-que-puede-salir-mal.html"><i class="fa fa-check"></i><b>7</b> RNA-seq en el mundo real y (casi) todo lo que puede salir mal</a></li>
<li class="chapter" data-level="8" data-path="an√°lisis-de-snrna-seq.html"><a href="an√°lisis-de-snrna-seq.html"><i class="fa fa-check"></i><b>8</b> An√°lisis de snRNA-seq</a></li>
<li class="chapter" data-level="9" data-path="revisi√≥n.html"><a href="revisi√≥n.html"><i class="fa fa-check"></i><b>9</b> Revisi√≥n</a>
<ul>
<li class="chapter" data-level="9.1" data-path="revisi√≥n.html"><a href="revisi√≥n.html#ejercicio-en-equipo"><i class="fa fa-check"></i><b>9.1</b> Ejercicio en equipo</a></li>
<li class="chapter" data-level="9.2" data-path="revisi√≥n.html"><a href="revisi√≥n.html#respuestas"><i class="fa fa-check"></i><b>9.2</b> Respuestas</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="rbioconductor-powered-team-data-science.html"><a href="rbioconductor-powered-team-data-science.html"><i class="fa fa-check"></i><b>10</b> R/Bioconductor-powered Team Data Science</a></li>
<li class="chapter" data-level="11" data-path="spatiallibd.html"><a href="spatiallibd.html"><i class="fa fa-check"></i><b>11</b> spatialLIBD</a></li>
<li class="chapter" data-level="12" data-path="comunidad-4.html"><a href="comunidad-4.html"><i class="fa fa-check"></i><b>12</b> Comunidad</a>
<ul>
<li class="chapter" data-level="12.1" data-path="comunidad-4.html"><a href="comunidad-4.html#de-alumnxs-lcg"><i class="fa fa-check"></i><b>12.1</b> De alumnxs LCG</a></li>
<li class="chapter" data-level="12.2" data-path="comunidad-4.html"><a href="comunidad-4.html#de-ustedes"><i class="fa fa-check"></i><b>12.2</b> De ustedes</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Intro RNA-seq LCG-UNAM 2026</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Intro RNA-seq LCG-UNAM 2026</h1>
<p class="author"><em>Leonardo Collado-Torres</em></p>
</div>
<div id="overview" class="section level1 unnumbered hasAnchor">
<h1>Overview<a href="index.html#overview" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Here you can find the files for the February 2026 introduction to R, RStudio and RNA-sequencing (RNA-seq) course for <a href="https://lcg.unam.mx/">LCG-UNAM</a> at <a href="https://ccg.unam.mx/">CCG-UNAM</a> (February 10-13, 2026). The rest of the chapters will be in Spanish.</p>
<p>Instructor:</p>
<ul>
<li><a href="http://lcolladotor.github.io/"><strong>Leonardo Collado-Torres</strong></a>: <a href="https://lcolladotor.github.io/bioc_team_ds/">team documentation website</a>, <a href="https://bsky.app/profile/lcolladotor.bsky.social">Bluesky</a></li>
</ul>
<p>Teaching assistants:</p>
<ul>
<li>Mar√≠a Jos√© Palma Mart√≠nez, <a href="">Bluesky</a>, <a href="">LinkedIn</a></li>
</ul>
<p>Guest presenters:</p>
<ul>
<li><a href="https://melii99.github.io/">Melissa May√©n Quiroz</a>, <a href="https://bsky.app/profile/meli-mq99.bsky.social">Bluesky</a></li>
<li><a href="https://www.gutierrezarceluslab.com/">Maria Guti√©rrez-Arcelus</a>, <a href="https://bsky.app/profile/mariagtac.bsky.social">Bluesky</a></li>
<li><a href="https://gabrielramirezv.github.io/">Gabriel Ram√≠rez-Vilchis</a>, <a href="https://bsky.app/profile/gabrielramirezv.bsky.social">Bluesky</a></li>
<li><a href="https://daianna-glez.github.io/">Daianna Gonz√°lez Padilla</a>, <a href="https://bsky.app/profile/daiannaglez.bsky.social">Bluesky</a></li>
</ul>
<!--
* [Renee Garc√≠a-Flores](https://reneegf.github.io/), [Bluesky](https://bsky.app/profile/freneegf.bsky.social)
-->
<p>Course Coordinators:</p>
<ul>
<li>Heladia Salgado</li>
<li>Julio Collado Vides</li>
</ul>
<p>Download the materials for this course with <code>usethis::use_course('lcolladotor/rnaseq_LCG-UNAM_2026')</code> or view online at <a href="http://lcolladotor.github.io/rnaseq_LCG-UNAM_2026"><strong>lcolladotor.github.io/rnaseq_LCG-UNAM_2026</strong></a>.</p>
<div id="code-of-conduct" class="section level2 unnumbered hasAnchor">
<h2>Code of Conduct<a href="index.html#code-of-conduct" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We‚Äôll follow the CDSB code of conduct <a href="https://comunidadbioinfo.github.io/codigo-de-conducta/">comunidadbioinfo.github.io/codigo-de-conducta/</a> as well as version 1.4.0 of the Bioconductor code of conduct <a href="https://bioconductor.github.io/bioc_coc_multilingual/">bioconductor.github.io/bioc_coc_multilingual/</a>.</p>
<p>For reporting any violations of the code of conduct, report them to the Instructor and/or Course Coordinators.</p>
</div>
<div id="course-schedule" class="section level2 unnumbered hasAnchor">
<h2>Course Schedule<a href="index.html#course-schedule" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><em>Local times in Cuernavaca, Mexico</em></p>
<ul>
<li>Tuesday February 10:
<ul>
<li>9-11 am: introduction to R, Positron, <em><a href="https://CRAN.R-project.org/package=usethis">usethis</a></em> and <em><a href="https://CRAN.R-project.org/package=here">here</a></em></li>
<li>12-2 pm: introduction to GitHub Copilot with Positron, and connecting Positron to GitHub</li>
</ul></li>
<li>Wednesday February 11:
<ul>
<li>9-11 am: introduction to Bioconductor</li>
<li>12-2 pm: expression data R/Bioconductor objects and visualization: <em><a href="https://bioconductor.org/packages/3.22/SummarizedExperiment">SummarizedExperiment</a></em> and <em><a href="https://bioconductor.org/packages/3.22/iSEE">iSEE</a></em>
<ul>
<li>12-1 pm: <em><a href="https://bioconductor.org/packages/3.22/SummarizedExperiment">SummarizedExperiment</a></em> intro</li>
<li>1-2 pm: <a href="https://melii99.github.io/">Melissa May√©n Quiroz</a> will teach you about <em><a href="https://bioconductor.org/packages/3.22/iSEE">iSEE</a></em> and showcase an <em><a href="https://bioconductor.org/packages/3.22/iSEE">iSEE</a></em> app she made for <a href="https://github.com/LieberInstitute/Hb_VTA_projector_profiling">LieberInstitute/Hb_VTA_projector_profiling</a>.</li>
</ul></li>
</ul></li>
<li>Thursday February 12:
<ul>
<li>9-11 am: making your own website with <em><a href="https://CRAN.R-project.org/package=postcards">postcards</a></em> by <a href="https://gabrielramirezv.github.io/">Gabriel Ram√≠rez-Vilchis</a></li>
<li>12-2 pm: downloading RNA-seq data using <em><a href="https://bioconductor.org/packages/3.22/recount3">recount3</a></em></li>
</ul></li>
<li>Friday February 13:
<ul>
<li>9-11 am: exploring statistical models with <em><a href="https://bioconductor.org/packages/3.22/ExploreModelMatrix">ExploreModelMatrix</a></em></li>
<li>12-1:30 pm: current examples of RNA-seq data analysis
<ul>
<li>Guest presentation by <a href="https://www.gutierrezarceluslab.com/">Maria Guti√©rrez-Arcelus</a></li>
<li>Guest presentation by <a href="https://daianna-glez.github.io/">Daianna Gonz√°lez Padilla</a></li>
<li>Guest presentation by <a href="https://gabrielramirezv.github.io/">Gabriel Ram√≠rez-Vilchis</a></li>
</ul></li>
<li>1:30-2 pm: differential expression analysis with <em><a href="https://bioconductor.org/packages/3.22/limma">limma</a></em>
<ul>
<li>review</li>
<li><em>R/Bioconductor-powered Team Data Science</em> overview
<ul>
<li>Potentially after 2 pm: open discussion (career paths, networking, etc)</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="external-links" class="section level2 unnumbered hasAnchor">
<h2>External links<a href="index.html#external-links" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li><a href="https://cursos.lcg.unam.mx/course/view.php?id=269#section-2">Cursos LCG</a></li>
<li><a href="https://github.com/lcolladotor/rnaseq_LCG-UNAM_2026">GitHub source code</a></li>
<li>Zoom (ask)</li>
<li><a href="http://132.248.220.108:8787/">LCG-UNAM RStudio server</a></li>
<li><a href="https://comunidadbioinfo.github.io/">CDSB</a></li>
<li><a href="https://comunidadbioinfo.slack.com">Slack CDSB Mexico</a>, in particular the <a href="https://comunidadbioinfo.slack.com/archives/C0AC8U6UPD2"><code>intro_rnaseq_lcg_2026</code> channel</a>.
<ul>
<li>You can join this Slack workspace through <a href="https://join.slack.com/t/comunidadbioinfo/shared_invite/zt-8lsvpm84-Fne1W0hadk6cpjgJS17Tnw">this link</a>.</li>
</ul></li>
<li><a href="https://community-bioc.zulipchat.com/">Bioconductor Zulip</a> community</li>
<li><a href="https://github.com/LieberInstitute/template_project">LieberInstitute/template_project</a></li>
<li>Example real project by <a href="https://daianna-glez.github.io/">Daianna Gonz√°lez Padilla</a>: <a href="https://github.com/LieberInstitute/smoking-nicotine-mouse">LieberInstitute/smoking-nicotine-mouse</a></li>
</ul>
</div>
<div id="course-prerequisites" class="section level2 unnumbered hasAnchor">
<h2>Course Prerequisites<a href="index.html#course-prerequisites" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>Install R 4.5.x from <a href="https://cran.r-project.org/">CRAN</a>.</li>
<li>Install <a href="https://positron.posit.co/download.html">Positron</a> which will be our Interactive Development Environment (IDE) of choice for the course.
<ul>
<li>Alternatively, you could install <a href="https://rstudio.com/products/rstudio/download/#download">RStudio IDE</a> version RStudio 2026.01.0 or newer.</li>
</ul></li>
<li>In either case, you will also need to install <a href="https://posit-dev.github.io/air/">Air</a> for automatic code formatting.</li>
<li>You will also have to requesst access to the <a href="https://github.com/education">GitHub Education</a> pack in order to use GitHub Copilot Pro ü§ñ.</li>
<li>Finally install the following R packages:</li>
</ul>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="index.html#cb1-1" tabindex="-1"></a><span class="do">## For installing Bioconductor packages</span></span>
<span id="cb1-2"><a href="index.html#cb1-2" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">requireNamespace</span>(<span class="st">&quot;BiocManager&quot;</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>)) {</span>
<span id="cb1-3"><a href="index.html#cb1-3" tabindex="-1"></a>    <span class="fu">install.packages</span>(<span class="st">&quot;BiocManager&quot;</span>)</span>
<span id="cb1-4"><a href="index.html#cb1-4" tabindex="-1"></a>}</span>
<span id="cb1-5"><a href="index.html#cb1-5" tabindex="-1"></a></span>
<span id="cb1-6"><a href="index.html#cb1-6" tabindex="-1"></a><span class="do">## Install required packages</span></span>
<span id="cb1-7"><a href="index.html#cb1-7" tabindex="-1"></a>BiocManager<span class="sc">::</span><span class="fu">install</span>(</span>
<span id="cb1-8"><a href="index.html#cb1-8" tabindex="-1"></a>    <span class="fu">c</span>(</span>
<span id="cb1-9"><a href="index.html#cb1-9" tabindex="-1"></a>        <span class="st">&quot;usethis&quot;</span>, <span class="do">## Utilities</span></span>
<span id="cb1-10"><a href="index.html#cb1-10" tabindex="-1"></a>        <span class="st">&quot;here&quot;</span>,</span>
<span id="cb1-11"><a href="index.html#cb1-11" tabindex="-1"></a>        <span class="st">&quot;biocthis&quot;</span>,</span>
<span id="cb1-12"><a href="index.html#cb1-12" tabindex="-1"></a>        <span class="st">&quot;lobstr&quot;</span>,</span>
<span id="cb1-13"><a href="index.html#cb1-13" tabindex="-1"></a>        <span class="st">&quot;postcards&quot;</span>,</span>
<span id="cb1-14"><a href="index.html#cb1-14" tabindex="-1"></a>        <span class="st">&quot;sessioninfo&quot;</span>,</span>
<span id="cb1-15"><a href="index.html#cb1-15" tabindex="-1"></a>        <span class="st">&quot;SummarizedExperiment&quot;</span>, <span class="do">## Main containers / vis</span></span>
<span id="cb1-16"><a href="index.html#cb1-16" tabindex="-1"></a>        <span class="st">&quot;iSEE&quot;</span>,</span>
<span id="cb1-17"><a href="index.html#cb1-17" tabindex="-1"></a>        <span class="st">&quot;edgeR&quot;</span>, <span class="do">## RNA-seq</span></span>
<span id="cb1-18"><a href="index.html#cb1-18" tabindex="-1"></a>        <span class="st">&quot;ExploreModelMatrix&quot;</span>,</span>
<span id="cb1-19"><a href="index.html#cb1-19" tabindex="-1"></a>        <span class="st">&quot;limma&quot;</span>,</span>
<span id="cb1-20"><a href="index.html#cb1-20" tabindex="-1"></a>        <span class="st">&quot;recount3&quot;</span>,</span>
<span id="cb1-21"><a href="index.html#cb1-21" tabindex="-1"></a>        <span class="st">&quot;pheatmap&quot;</span>, <span class="do">## Visualization</span></span>
<span id="cb1-22"><a href="index.html#cb1-22" tabindex="-1"></a>        <span class="st">&quot;ggplot2&quot;</span>,</span>
<span id="cb1-23"><a href="index.html#cb1-23" tabindex="-1"></a>        <span class="st">&quot;patchwork&quot;</span>,</span>
<span id="cb1-24"><a href="index.html#cb1-24" tabindex="-1"></a>        <span class="st">&quot;RColorBrewer&quot;</span>,</span>
<span id="cb1-25"><a href="index.html#cb1-25" tabindex="-1"></a>        <span class="st">&quot;ComplexHeatmap&quot;</span>,</span>
<span id="cb1-26"><a href="index.html#cb1-26" tabindex="-1"></a>        <span class="st">&quot;spatialLIBD&quot;</span> <span class="do">## Advanced</span></span>
<span id="cb1-27"><a href="index.html#cb1-27" tabindex="-1"></a>    )</span>
<span id="cb1-28"><a href="index.html#cb1-28" tabindex="-1"></a>)</span></code></pre></div>
<div id="getting-set-up" class="section level3 unnumbered hasAnchor">
<h3>Getting set up<a href="index.html#getting-set-up" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Here are more step by step instructions for the course prerequisites.</p>
<p>You must install <a href="https://cran.r-project.org">R</a> and <a href="https://positron.posit.co/download.html">Positron</a> (available from <a href="https://posit.co/">Posit</a>) on your computing environment. As an alternative to Positron, you can install <a href="https://posit.co/products/open-source/rstudio/?sid=1">RStudio IDE</a>.</p>
<p>These are three <strong>different</strong> applications that must be installed separately before they can be used together:</p>
<ul>
<li><p><code>R</code> is the core underlying programming language and computing engine that we will be learning in this course</p></li>
<li><p><code>RStudio IDE</code> is an interface into R that makes many aspects of using and programming R simpler</p></li>
<li><p><code>Positron</code> is a new interface for working with R developed by <a href="https://posit.co/">Posit</a> (formerly RStudio). All new features are being implemented in <code>Positron</code>, but as it was just recently released, it might have some bugs. If you are choosing between <code>RStudio IDE</code> and <code>Positron</code>, check <a href="https://positron.posit.co/faqs.html#is-rstudio-going-away" class="uri">https://positron.posit.co/faqs.html#is-rstudio-going-away</a>.</p></li>
</ul>
<p><code>R</code>, <code>RStudio IDE</code>, and <code>Positron</code> are available for Windows, macOS, and most flavors of Unix and Linux. Please download the version that is suitable for your computing setup.</p>
<p>Throughout the course, we will make use of numerous R add-on packages that must be installed over the Internet. Packages can be installed using the <code>install.packages()</code> function in R. For example, to install the <code>tidyverse</code> package, you can run</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="index.html#cb2-1" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">&quot;tidyverse&quot;</span>)</span></code></pre></div>
<p>in the R console.</p>
<p>We will also use <a href="https://git-scm.com/">Git</a> to version control your code, which you can install following the <a href="https://happygitwithr.com/"><em>Happy Git and GitHub for the useR</em></a> book. Through the <a href="https://github.com/education">GitHub Education</a> pack we will get free access to <a href="https://github.com/features/copilot">GittHub Copilot Pro</a> to help us write code: think about it as an enhanced auto-complete. Finally, we‚Äôll use <a href="https://posit-dev.github.io/air/">Air</a> for automatically formatting our R code.</p>
</div>
<div id="how-to-download-r-for-windows" class="section level3 unnumbered hasAnchor">
<h3>How to Download R for Windows<a href="index.html#how-to-download-r-for-windows" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Go to <a href="https://cran.r-project.org" class="uri">https://cran.r-project.org</a> and</p>
<ol style="list-style-type: decimal">
<li><p>Click the link to ‚ÄúDownload R for Windows‚Äù</p></li>
<li><p>Click on ‚Äúbase‚Äù</p></li>
<li><p>Click on ‚ÄúDownload R 4.5.2 for Windows‚Äù</p></li>
</ol>
<div class="float">
<img src="https://lcolladotor.github.io/jhustatcomputing/videos/downloadRWindows.gif" style="width:80.0%" alt="Video Demo for Downloading R for Windows" />
<div class="figcaption">Video Demo for Downloading R for Windows</div>
</div>
<p><em>The version in the video is not the latest version of R. Please download the latest version.</em></p>
</div>
<div id="how-to-download-r-for-the-mac" class="section level3 unnumbered hasAnchor">
<h3>How to Download R for the Mac<a href="index.html#how-to-download-r-for-the-mac" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Go to <a href="https://cran.r-project.org" class="uri">https://cran.r-project.org</a> and</p>
<ol style="list-style-type: decimal">
<li><p>Click the link to ‚ÄúDownload R for (Mac) OS X‚Äù.</p></li>
<li><p>Click on ‚ÄúR-4.5.2-arm64.pkg‚Äù (for Apple Silicon Macs)</p></li>
</ol>
<div class="float">
<img src="https://lcolladotor.github.io/jhustatcomputing/videos/downloadRMac.gif" style="width:80.0%" alt="Video Demo for Downloading R for the Mac" />
<div class="figcaption">Video Demo for Downloading R for the Mac</div>
</div>
<p><em>The version in the video is not the latest version of R. Please download the latest version.</em></p>
</div>
<div id="how-to-download-rstudio-ide" class="section level3 unnumbered hasAnchor">
<h3>How to Download <code>RStudio IDE</code><a href="index.html#how-to-download-rstudio-ide" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Go to <a href="https://posit.co/" class="uri">https://posit.co/</a> (formerly <a href="">https://rstudio.com</a>) and</p>
<ol style="list-style-type: decimal">
<li><p>Click on ‚ÄúFree &amp; Open Source‚Äù in the top menu</p></li>
<li><p>Click on ‚ÄúRStudio IDE‚Äù in the drop down menu</p></li>
<li><p>Click the button that says ‚ÄúDOWNLOAD RSTUDIO DESKTOP‚Äù</p></li>
<li><p>Click the button under ‚ÄúDownload RStudio‚Äù which will take you to <a href="https://posit.co/download/rstudio-desktop/" class="uri">https://posit.co/download/rstudio-desktop/</a></p></li>
<li><p>Under the section ‚ÄúAll Installers and Tarballs‚Äù choose the file that is appropriate for your operating system.</p></li>
</ol>
<div class="float">
<img src="https://lcolladotor.github.io/jhustatcomputing/videos/downloadRStudio.gif" style="width:80.0%" alt="Video Demo for Downloading RStudio" />
<div class="figcaption">Video Demo for Downloading RStudio</div>
</div>
<p><em>NOTE: The video shows how to download RStudio for the Mac but you should download RStudio for whatever computing setup you have.</em></p>
</div>
<div id="how-to-download-positron" class="section level3 unnumbered hasAnchor">
<h3>How to Download Positron<a href="index.html#how-to-download-positron" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Go to <a href="https://posit.co/" class="uri">https://posit.co/</a> and</p>
<ol style="list-style-type: decimal">
<li><p>Click on ‚ÄúFree &amp; Open Source‚Äù in the top menu</p></li>
<li><p>Click on ‚ÄúPositron‚Äù in the drop down menu</p></li>
<li><p>Click on ‚ÄúDownload Positron‚Äù which will take you to <a href="https://positron.posit.co/download.html" class="uri">https://positron.posit.co/download.html</a></p></li>
<li><p>Read the license agreement.</p></li>
<li><p>Accept the license agreement.</p></li>
<li><p>On the table with all the download options, choose the file that is appropriate for your operating system.</p></li>
</ol>
</div>
<div id="how-to-download-git" class="section level3 unnumbered hasAnchor">
<h3>How to Download Git<a href="index.html#how-to-download-git" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Install <code>Git</code> on your computer following the detailed instructions at <a href="https://happygitwithr.com/install-git" class="uri">https://happygitwithr.com/install-git</a>, which will depend on your operating system.</p>
</div>
<div id="create-your-github-account" class="section level3 unnumbered hasAnchor">
<h3>Create your GitHub account<a href="index.html#create-your-github-account" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Create a personal GitHub account following the instructions at <a href="https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account" class="uri">https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account</a>.</p>
<p>Following <a href="https://bsky.app/profile/jtleek.bsky.social">Jeff Leek</a>‚Äôs advice on <a href="https://leanpub.com/modernscientist"><strong>How to be a modern scientist</strong></a> that was part of my team‚Äôs bootcamps at <a href="https://lcolladotor.github.io/bioc_team_ds/how-to-be-a-modern-scientist.html" class="uri">https://lcolladotor.github.io/bioc_team_ds/how-to-be-a-modern-scientist.html</a>, try to choose a username that will be the same one you use on your email and other work-related social media platforms such as Bluesky, LinkedIn, etc.</p>
<iframe width="560" height="315" src="https://www.youtube.com/embed/93hA-iPWsbs?si=o-Ekbw7iOSoa5j6e" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen>
</iframe>
<p>If you are using <code>Positron</code>, under the ‚ÄúAccounts‚Äù options (bottom left corner), click on ‚ÄúSign in with GitHub <span class="math display">\[...\]</span>‚Äù, provide the 8 digit code, and authorize ‚Äúposit-dev‚Äù.</p>
</div>
<div id="how-to-apply-for-github-education-and-configure-github-copilot" class="section level3 unnumbered hasAnchor">
<h3>How to Apply for GitHub Education and configure GitHub Copilot<a href="index.html#how-to-apply-for-github-education-and-configure-github-copilot" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Go to <a href="https://github.com/education" class="uri">https://github.com/education</a> and</p>
<ol style="list-style-type: decimal">
<li>Click on ‚ÄúJoin GitHub education‚Äù (likely <a href="https://github.com/settings/education/benefits?locale=en-US" class="uri">https://github.com/settings/education/benefits?locale=en-US</a>)</li>
<li>Sign in to your GitHub account if you haven‚Äôt logged in</li>
<li>Click on ‚ÄúStart Application‚Äù</li>
<li>There will be a few options to verify that you are a student. One of them is by having a photo of your student ID with dates showing that your student ID is still valid. I‚Äôve used a screenshot of my Faculty website at <a href="https://publichealth.jhu.edu/faculty/4647/leonardo-collado-torres" class="uri">https://publichealth.jhu.edu/faculty/4647/leonardo-collado-torres</a> when validating my information.</li>
<li>Wait a few <strong>days</strong> for GitHub to process your application.</li>
<li>Once your application has been approved, if you are using <code>RStudio IDE</code> follow the instructions at <a href="https://docs.posit.co/ide/user/ide/guide/tools/copilot.html" class="uri">https://docs.posit.co/ide/user/ide/guide/tools/copilot.html</a> to enable GitHub Copilot in RStudio IDE. I have the ‚ÄúIndex project files with GitHub Copilot‚Äù checkbox enabled on my preferences.</li>
<li>If you are using <code>Positron</code>, follow the instructions at <a href="https://positron.posit.co/assistant" class="uri">https://positron.posit.co/assistant</a> to enable ‚ÄúPositron assistant‚Äù. Then under the ‚ÄúPositron Assistant‚Äù chat, click on ‚ÄúAdd Anthropic as a Chat Provider‚Äù, then select ‚ÄúGitHub Copilot‚Äù and click ‚ÄúSign in‚Äù.</li>
</ol>
</div>
<div id="how-to-get-air-auto-code-formatter" class="section level3 unnumbered hasAnchor">
<h3>How to get Air (auto-code formatter)<a href="index.html#how-to-get-air-auto-code-formatter" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Go to <a href="https://posit-dev.github.io/air/" class="uri">https://posit-dev.github.io/air/</a> and</p>
<ol style="list-style-type: decimal">
<li><p>Click on ‚ÄúRStudio‚Äù if you are using <code>RStudio IDE</code>. That will take you to <a href="https://posit-dev.github.io/air/editor-rstudio.html" class="uri">https://posit-dev.github.io/air/editor-rstudio.html</a> where you will find the detailed installation instructions.</p>
<ul>
<li>Note that I recommend enabling the <code>Tools -&gt; Global Options -&gt; Code -&gt; Saving</code> and check <code>Reformat documents on save</code> option. Particularly if you are working on your own code.</li>
</ul></li>
<li><p>Click on ‚ÄúVS Code and Positron‚Äù if you are using <code>Positron</code>. That will take you to <a href="https://posit-dev.github.io/air/editor-vscode.html" class="uri">https://posit-dev.github.io/air/editor-vscode.html</a> where some options and configuration steps are described. Note however, that <code>Air</code> is already available from <code>Positron</code>. You will need to open your <code>settings.json</code> file from the Command Palette (<code>Shift + Cmd + P</code> on Mac/Linux, <code>Shift + Ctrl + P</code> on Windows):</p>
<ul>
<li><p>Run <code>Preferences: Open Workspace Settings (JSON)</code> to modify workspace specific settings (recommended).</p></li>
<li><p>Run <code>Preferences: Open User Settings (JSON)</code> to modify global user settings (preferred option if you are not working with code from others). Then edit your <code>settings.json</code> with the relevant lines about R code formatting and <code>Air</code> as shown below:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode json"><code class="sourceCode json"><span id="cb3-1"><a href="index.html#cb3-1" tabindex="-1"></a><span class="fu">{</span></span>
<span id="cb3-2"><a href="index.html#cb3-2" tabindex="-1"></a>    <span class="dt">&quot;files.associations&quot;</span><span class="fu">:</span> <span class="fu">{</span></span>
<span id="cb3-3"><a href="index.html#cb3-3" tabindex="-1"></a>        <span class="dt">&quot;renv.lock&quot;</span><span class="fu">:</span> <span class="st">&quot;json&quot;</span></span>
<span id="cb3-4"><a href="index.html#cb3-4" tabindex="-1"></a>    <span class="fu">},</span></span>
<span id="cb3-5"><a href="index.html#cb3-5" tabindex="-1"></a>    <span class="dt">&quot;positron.assistant.gitIntegration.enable&quot;</span><span class="fu">:</span> <span class="kw">true</span><span class="fu">,</span></span>
<span id="cb3-6"><a href="index.html#cb3-6" tabindex="-1"></a>    <span class="dt">&quot;positron.assistant.showTokenUsage.enable&quot;</span><span class="fu">:</span> <span class="kw">true</span><span class="fu">,</span></span>
<span id="cb3-7"><a href="index.html#cb3-7" tabindex="-1"></a>    <span class="dt">&quot;positron.assistant.toolDetails.enable&quot;</span><span class="fu">:</span> <span class="kw">true</span><span class="fu">,</span></span>
<span id="cb3-8"><a href="index.html#cb3-8" tabindex="-1"></a>    <span class="dt">&quot;[r]&quot;</span><span class="fu">:</span> <span class="fu">{</span></span>
<span id="cb3-9"><a href="index.html#cb3-9" tabindex="-1"></a>        <span class="dt">&quot;editor.formatOnSave&quot;</span><span class="fu">:</span> <span class="kw">true</span><span class="fu">,</span></span>
<span id="cb3-10"><a href="index.html#cb3-10" tabindex="-1"></a>        <span class="dt">&quot;editor.defaultFormatter&quot;</span><span class="fu">:</span> <span class="st">&quot;Posit.air-vscode&quot;</span></span>
<span id="cb3-11"><a href="index.html#cb3-11" tabindex="-1"></a>    <span class="fu">}</span></span>
<span id="cb3-12"><a href="index.html#cb3-12" tabindex="-1"></a><span class="fu">}</span></span></code></pre></div></li>
</ul>
<ol style="list-style-type: decimal">
<li>You can also configure a few options at <a href="positron://settings/air.dependencyLogLevels">positron://settings/air.dependencyLogLevels</a> and related settings.</li>
</ol></li>
</ol>
</div>
</div>
<div id="course-evaluation" class="section level2 unnumbered hasAnchor">
<h2>Course Evaluation<a href="index.html#course-evaluation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>Participation
<ul>
<li>(main evidence) written and/or verbal participation during class
<ul>
<li>This is tracked by day by the TA. Each day counts as a category.</li>
</ul></li>
<li>(optional, strongly encouraged for your own career) Community building: CDSB Slack (introduce yourself), Bluesky, GitHub, etc. These are tracked by the TA: please let the TA know of your participations.
<ul>
<li>GitHub issues on the course repository</li>
<li>CDSB Slack <a href="https://comunidadbioinfo.slack.com/archives/CAB6CHXUY">#bienvenida</a> channel</li>
<li>Bioconductor Zulip <a href="https://community-bioc.zulipchat.com/#narrow/channel/507510-introductions">#introductions</a> channel</li>
<li>Bluesky posts related to the course</li>
</ul></li>
<li>Grading:
<ul>
<li>0%: no participation</li>
<li>25%: participation in only 1 out of 8 categories (4 days, 4 public participation categories)</li>
<li>50%: participation in 2 out of 8 categories</li>
<li>75%: participation in 3 out of 8 categories</li>
<li>100%: participation in 4 or more categories</li>
</ul></li>
</ul></li>
<li>Public work
<ul>
<li>(main evidence) Taking notes on your GitHub class notes repositories</li>
<li>(optional) Deploy an iSEE <code>shiny</code> app on shinyapps.io</li>
<li>(advanced and optional) R package with notes following <code>usethis::create_package()</code> + <code>biocthis::use_bioc_pkg_templates()</code>.</li>
</ul></li>
<li>Final project
<ul>
<li>(main evidence) code in a public GitHub repository (different from the notes one) following the structure from <a href="https://github.com/LieberInstitute/template_project">LieberInstitute/template_project</a></li>
<li>(optional) a rendered Rmd file (RPubs, GitHub, etc) on a public URL</li>
<li>(optional advanced) Copy this repository, remove all the Rmds except the index.Rmd and edit accordingly. GitHub Actions will then help you render it. You‚Äôll need to fetch the <code>gh-pages</code> branch and at least make one commit for the HTML to be available such as this <a href="https://github.com/lcolladotor/rnaseq_LCG-UNAM_2021/commit/e15bd6c234dd0ca4233050a4a0a42184994fef3b">commit</a>. You might also run into <a href="https://github.com/ad-m/github-push-action/issues/96#issuecomment-1396347833">this GitHub default permission setting</a> that you‚Äôll need to change.</li>
</ul></li>
</ul>
<p>The final grade will be a weighted average from the evaluation by the TA (20%), your public work (40%) and your final project (40%).</p>
<p>Projects, public work and notes are due at 9 am US Eastern on Monday February 23rd, though we recommend doing any work related to this course by Friday February 20th.</p>
</div>
<div id="r-session-information" class="section level2 unnumbered hasAnchor">
<h2>R session information<a href="index.html#r-session-information" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Details on the R version used for making this book. The source code is available at <a href="https://github.com/lcolladotor/rnaseq_LCG-UNAM_2026"><code>lcolladotor/rnaseq_LCG-UNAM_2026</code></a>.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="index.html#cb4-1" tabindex="-1"></a><span class="do">## Load the package at the top of your script</span></span>
<span id="cb4-2"><a href="index.html#cb4-2" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;sessioninfo&quot;</span>)</span>
<span id="cb4-3"><a href="index.html#cb4-3" tabindex="-1"></a></span>
<span id="cb4-4"><a href="index.html#cb4-4" tabindex="-1"></a><span class="do">##  Utilities</span></span>
<span id="cb4-5"><a href="index.html#cb4-5" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;BiocStyle&quot;</span>)</span>
<span id="cb4-6"><a href="index.html#cb4-6" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;biocthis&quot;</span>)</span>
<span id="cb4-7"><a href="index.html#cb4-7" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;here&quot;</span>)</span>
<span id="cb4-8"><a href="index.html#cb4-8" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;lobstr&quot;</span>)</span>
<span id="cb4-9"><a href="index.html#cb4-9" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;postcards&quot;</span>)</span>
<span id="cb4-10"><a href="index.html#cb4-10" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;usethis&quot;</span>)</span>
<span id="cb4-11"><a href="index.html#cb4-11" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;sessioninfo&quot;</span>)</span>
<span id="cb4-12"><a href="index.html#cb4-12" tabindex="-1"></a></span>
<span id="cb4-13"><a href="index.html#cb4-13" tabindex="-1"></a><span class="do">## Main containers / vis</span></span>
<span id="cb4-14"><a href="index.html#cb4-14" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;SummarizedExperiment&quot;</span>)</span>
<span id="cb4-15"><a href="index.html#cb4-15" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;iSEE&quot;</span>)</span>
<span id="cb4-16"><a href="index.html#cb4-16" tabindex="-1"></a></span>
<span id="cb4-17"><a href="index.html#cb4-17" tabindex="-1"></a><span class="do">## RNA-seq</span></span>
<span id="cb4-18"><a href="index.html#cb4-18" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;edgeR&quot;</span>)</span>
<span id="cb4-19"><a href="index.html#cb4-19" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ExploreModelMatrix&quot;</span>)</span>
<span id="cb4-20"><a href="index.html#cb4-20" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;limma&quot;</span>)</span>
<span id="cb4-21"><a href="index.html#cb4-21" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;recount3&quot;</span>)</span>
<span id="cb4-22"><a href="index.html#cb4-22" tabindex="-1"></a></span>
<span id="cb4-23"><a href="index.html#cb4-23" tabindex="-1"></a><span class="do">## Visualization</span></span>
<span id="cb4-24"><a href="index.html#cb4-24" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ComplexHeatmap&quot;</span>)</span>
<span id="cb4-25"><a href="index.html#cb4-25" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggplot2&quot;</span>)</span>
<span id="cb4-26"><a href="index.html#cb4-26" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;patchwork&quot;</span>)</span>
<span id="cb4-27"><a href="index.html#cb4-27" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;pheatmap&quot;</span>)</span>
<span id="cb4-28"><a href="index.html#cb4-28" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;RColorBrewer&quot;</span>)</span>
<span id="cb4-29"><a href="index.html#cb4-29" tabindex="-1"></a></span>
<span id="cb4-30"><a href="index.html#cb4-30" tabindex="-1"></a><span class="do">## Advanced</span></span>
<span id="cb4-31"><a href="index.html#cb4-31" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;spatialLIBD&quot;</span>)</span></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="index.html#cb5-1" tabindex="-1"></a><span class="do">## Reproducibility information</span></span>
<span id="cb5-2"><a href="index.html#cb5-2" tabindex="-1"></a><span class="fu">options</span>(<span class="at">width =</span> <span class="dv">120</span>)</span>
<span id="cb5-3"><a href="index.html#cb5-3" tabindex="-1"></a><span class="fu">session_info</span>()</span></code></pre></div>
<pre><code>## ‚îÄ Session info ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ
##  setting  value
##  version  R version 4.5.2 (2025-10-31)
##  os       Ubuntu 24.04.3 LTS
##  system   x86_64, linux-gnu
##  ui       X11
##  language (EN)
##  collate  en_US.UTF-8
##  ctype    en_US.UTF-8
##  tz       UTC
##  date     2026-02-11
##  pandoc   3.8.2.1 @ /usr/bin/ (via rmarkdown)
##  quarto   1.7.32 @ /usr/local/bin/quarto
## 
## ‚îÄ Packages ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ
##  package              * version   date (UTC) lib source
##  abind                  1.4-8     2024-09-12 [1] RSPM (R 4.5.0)
##  AnnotationDbi          1.72.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  AnnotationHub          4.0.0     2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  attempt                0.3.1     2020-05-03 [1] RSPM (R 4.5.0)
##  beachmat               2.26.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  beeswarm               0.4.0     2021-06-01 [1] RSPM (R 4.5.0)
##  benchmarkme            1.0.8     2022-06-12 [1] RSPM (R 4.5.0)
##  benchmarkmeData        2.0.0     2026-01-19 [1] RSPM (R 4.5.0)
##  Biobase              * 2.70.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  BiocFileCache          3.0.0     2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  BiocGenerics         * 0.56.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  BiocIO                 1.20.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  BiocManager            1.30.27   2025-11-14 [2] CRAN (R 4.5.2)
##  BiocNeighbors          2.4.0     2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  BiocParallel           1.44.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  BiocSingular           1.26.1    2025-11-17 [1] Bioconductor 3.22 (R 4.5.2)
##  BiocStyle            * 2.38.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  biocthis             * 1.20.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  BiocVersion            3.22.0    2025-10-07 [2] Bioconductor 3.22 (R 4.5.2)
##  Biostrings             2.78.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  bit                    4.6.0     2025-03-06 [1] RSPM (R 4.5.0)
##  bit64                  4.6.0-1   2025-01-16 [1] RSPM (R 4.5.0)
##  bitops                 1.0-9     2024-10-03 [1] RSPM (R 4.5.0)
##  blob                   1.3.0     2026-01-14 [1] RSPM (R 4.5.0)
##  bookdown               0.46      2025-12-05 [1] RSPM (R 4.5.0)
##  bslib                  0.10.0    2026-01-26 [2] RSPM (R 4.5.0)
##  cachem                 1.1.0     2024-05-16 [2] RSPM (R 4.5.0)
##  cigarillo              1.0.0     2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  circlize               0.4.17    2025-12-08 [1] RSPM (R 4.5.0)
##  cli                    3.6.5     2025-04-23 [2] RSPM (R 4.5.0)
##  clue                   0.3-66    2024-11-13 [1] RSPM (R 4.5.0)
##  cluster                2.1.8.2   2026-02-05 [3] RSPM (R 4.5.0)
##  codetools              0.2-20    2024-03-31 [3] CRAN (R 4.5.2)
##  colorspace             2.1-2     2025-09-22 [1] RSPM (R 4.5.0)
##  colourpicker           1.3.0     2023-08-21 [1] RSPM (R 4.5.0)
##  ComplexHeatmap       * 2.26.1    2026-02-03 [1] Bioconductor 3.22 (R 4.5.2)
##  config                 0.3.2     2023-08-30 [1] RSPM (R 4.5.0)
##  cowplot                1.2.0     2025-07-07 [1] RSPM (R 4.5.0)
##  crayon                 1.5.3     2024-06-20 [2] RSPM (R 4.5.0)
##  curl                   7.0.0     2025-08-19 [2] RSPM (R 4.5.0)
##  data.table             1.18.2.1  2026-01-27 [1] RSPM (R 4.5.0)
##  DBI                    1.2.3     2024-06-02 [1] RSPM (R 4.5.0)
##  dbplyr                 2.5.1     2025-09-10 [1] RSPM (R 4.5.0)
##  DelayedArray           0.36.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  digest                 0.6.39    2025-11-19 [2] RSPM (R 4.5.0)
##  doParallel             1.0.17    2022-02-07 [1] RSPM (R 4.5.0)
##  dplyr                  1.2.0     2026-02-03 [1] RSPM (R 4.5.0)
##  DT                     0.34.0    2025-09-02 [1] RSPM (R 4.5.0)
##  edgeR                * 4.8.2     2025-12-25 [1] Bioconductor 3.22 (R 4.5.2)
##  evaluate               1.0.5     2025-08-27 [2] RSPM (R 4.5.0)
##  ExperimentHub          3.0.0     2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  ExploreModelMatrix   * 1.22.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  farver                 2.1.2     2024-05-13 [1] RSPM (R 4.5.0)
##  fastmap                1.2.0     2024-05-15 [2] RSPM (R 4.5.0)
##  filelock               1.0.3     2023-12-11 [1] RSPM (R 4.5.0)
##  foreach                1.5.2     2022-02-02 [1] RSPM (R 4.5.0)
##  fs                     1.6.6     2025-04-12 [2] RSPM (R 4.5.0)
##  generics             * 0.1.4     2025-05-09 [1] RSPM (R 4.5.0)
##  GenomicAlignments      1.46.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  GenomicRanges        * 1.62.1    2025-12-08 [1] Bioconductor 3.22 (R 4.5.2)
##  GetoptLong             1.1.0     2025-11-28 [1] RSPM (R 4.5.0)
##  ggbeeswarm             0.7.3     2025-11-29 [1] RSPM (R 4.5.0)
##  ggplot2              * 4.0.2     2026-02-03 [1] RSPM (R 4.5.0)
##  ggrepel                0.9.6     2024-09-07 [1] RSPM (R 4.5.0)
##  GlobalOptions          0.1.3     2025-11-28 [1] RSPM (R 4.5.0)
##  glue                   1.8.0     2024-09-30 [2] RSPM (R 4.5.0)
##  golem                  0.5.1     2024-08-27 [1] RSPM (R 4.5.0)
##  gridExtra              2.3       2017-09-09 [1] RSPM (R 4.5.0)
##  gtable                 0.3.6     2024-10-25 [1] RSPM (R 4.5.0)
##  here                 * 1.0.2     2025-09-15 [1] RSPM (R 4.5.0)
##  htmltools              0.5.9     2025-12-04 [2] RSPM (R 4.5.0)
##  htmlwidgets            1.6.4     2023-12-06 [2] RSPM (R 4.5.0)
##  httpuv                 1.6.16    2025-04-16 [2] RSPM (R 4.5.0)
##  httr                   1.4.7     2023-08-15 [1] RSPM (R 4.5.0)
##  httr2                  1.2.2     2025-12-08 [2] RSPM (R 4.5.0)
##  igraph                 2.2.1     2025-10-27 [1] RSPM (R 4.5.0)
##  IRanges              * 2.44.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  irlba                  2.3.7     2026-01-30 [1] RSPM (R 4.5.0)
##  iSEE                 * 2.22.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  iterators              1.0.14    2022-02-05 [1] RSPM (R 4.5.0)
##  jquerylib              0.1.4     2021-04-26 [2] RSPM (R 4.5.0)
##  jsonlite               2.0.0     2025-03-27 [2] RSPM (R 4.5.0)
##  KEGGREST               1.50.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  knitr                  1.51      2025-12-20 [2] RSPM (R 4.5.0)
##  later                  1.4.5     2026-01-08 [2] RSPM (R 4.5.0)
##  lattice                0.22-9    2026-02-09 [3] RSPM (R 4.5.0)
##  lazyeval               0.2.2     2019-03-15 [1] RSPM (R 4.5.0)
##  lifecycle              1.0.5     2026-01-08 [1] RSPM (R 4.5.0)
##  limma                * 3.66.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  listviewer             4.0.0     2023-09-30 [1] RSPM (R 4.5.0)
##  lobstr               * 1.1.3     2025-11-14 [1] RSPM (R 4.5.0)
##  locfit                 1.5-9.12  2025-03-05 [1] RSPM (R 4.5.0)
##  magick                 2.9.0     2025-09-08 [1] RSPM (R 4.5.0)
##  magrittr               2.0.4     2025-09-12 [2] RSPM (R 4.5.0)
##  MASS                   7.3-65    2025-02-28 [3] CRAN (R 4.5.2)
##  Matrix                 1.7-4     2025-08-28 [3] CRAN (R 4.5.2)
##  MatrixGenerics       * 1.22.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  matrixStats          * 1.5.0     2025-01-07 [1] RSPM (R 4.5.0)
##  memoise                2.0.1     2021-11-26 [2] RSPM (R 4.5.0)
##  mgcv                   1.9-4     2025-11-07 [3] RSPM (R 4.5.0)
##  mime                   0.13      2025-03-17 [2] RSPM (R 4.5.0)
##  miniUI                 0.1.2     2025-04-17 [2] RSPM (R 4.5.0)
##  nlme                   3.1-168   2025-03-31 [3] CRAN (R 4.5.2)
##  otel                   0.2.0     2025-08-29 [2] RSPM (R 4.5.0)
##  paletteer              1.7.0     2026-01-08 [1] RSPM (R 4.5.0)
##  patchwork            * 1.3.2     2025-08-25 [1] RSPM (R 4.5.0)
##  pheatmap             * 1.0.13    2025-06-05 [1] RSPM (R 4.5.0)
##  pillar                 1.11.1    2025-09-17 [2] RSPM (R 4.5.0)
##  pkgconfig              2.0.3     2019-09-22 [2] RSPM (R 4.5.0)
##  plotly                 4.12.0    2026-01-24 [1] RSPM (R 4.5.0)
##  png                    0.1-8     2022-11-29 [1] RSPM (R 4.5.0)
##  postcards            * 0.2.3     2022-01-07 [1] RSPM (R 4.5.0)
##  promises               1.5.0     2025-11-01 [2] RSPM (R 4.5.0)
##  purrr                  1.2.1     2026-01-09 [2] RSPM (R 4.5.0)
##  R.cache                0.17.0    2025-05-02 [1] RSPM (R 4.5.0)
##  R.methodsS3            1.8.2     2022-06-13 [1] RSPM (R 4.5.0)
##  R.oo                   1.27.1    2025-05-02 [1] RSPM (R 4.5.0)
##  R.utils                2.13.0    2025-02-24 [1] RSPM (R 4.5.0)
##  R6                     2.6.1     2025-02-15 [2] RSPM (R 4.5.0)
##  rappdirs               0.3.4     2026-01-17 [2] RSPM (R 4.5.0)
##  RColorBrewer         * 1.1-3     2022-04-03 [1] RSPM (R 4.5.0)
##  Rcpp                   1.1.1     2026-01-10 [2] RSPM (R 4.5.0)
##  RCurl                  1.98-1.17 2025-03-22 [1] RSPM (R 4.5.0)
##  recount3             * 1.20.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  rematch2               2.1.2     2020-05-01 [1] RSPM (R 4.5.0)
##  restfulr               0.0.16    2025-06-27 [1] RSPM (R 4.5.2)
##  rintrojs               0.3.4     2024-01-11 [1] RSPM (R 4.5.0)
##  rjson                  0.2.23    2024-09-16 [1] RSPM (R 4.5.0)
##  rlang                  1.1.7     2026-01-09 [1] RSPM (R 4.5.0)
##  rmarkdown              2.30      2025-09-28 [2] RSPM (R 4.5.0)
##  rprojroot              2.1.1     2025-08-26 [2] RSPM (R 4.5.0)
##  Rsamtools              2.26.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  RSQLite                2.4.6     2026-02-06 [1] RSPM (R 4.5.0)
##  rstudioapi             0.18.0    2026-01-16 [2] RSPM (R 4.5.0)
##  rsvd                   1.0.5     2021-04-16 [1] RSPM (R 4.5.0)
##  rtracklayer            1.70.1    2025-12-22 [1] Bioconductor 3.22 (R 4.5.2)
##  S4Arrays               1.10.1    2025-12-01 [1] Bioconductor 3.22 (R 4.5.2)
##  S4Vectors            * 0.48.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  S7                     0.2.1     2025-11-14 [1] RSPM (R 4.5.0)
##  sass                   0.4.10    2025-04-11 [2] RSPM (R 4.5.0)
##  ScaledMatrix           1.18.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  scales                 1.4.0     2025-04-24 [1] RSPM (R 4.5.0)
##  scater                 1.38.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  scuttle                1.20.0    2025-10-30 [1] Bioconductor 3.22 (R 4.5.2)
##  Seqinfo              * 1.0.0     2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  sessioninfo          * 1.2.3     2025-02-05 [2] RSPM (R 4.5.0)
##  shape                  1.4.6.1   2024-02-23 [1] RSPM (R 4.5.0)
##  shiny                  1.12.1    2025-12-09 [2] RSPM (R 4.5.0)
##  shinyAce               0.4.4     2025-02-03 [1] RSPM (R 4.5.0)
##  shinydashboard         0.7.3     2025-04-21 [1] RSPM (R 4.5.0)
##  shinyjs                2.1.1     2026-01-15 [1] RSPM (R 4.5.0)
##  shinyWidgets           0.9.0     2025-02-21 [1] RSPM (R 4.5.0)
##  SingleCellExperiment * 1.32.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  SparseArray            1.10.8    2025-12-18 [1] Bioconductor 3.22 (R 4.5.2)
##  SpatialExperiment    * 1.20.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  spatialLIBD          * 1.22.0    2025-11-04 [1] Bioconductor 3.22 (R 4.5.2)
##  statmod                1.5.1     2025-10-09 [1] RSPM (R 4.5.0)
##  styler                 1.11.0    2025-10-13 [1] RSPM (R 4.5.0)
##  SummarizedExperiment * 1.40.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  tibble                 3.3.1     2026-01-11 [2] RSPM (R 4.5.0)
##  tidyr                  1.3.2     2025-12-19 [1] RSPM (R 4.5.0)
##  tidyselect             1.2.1     2024-03-11 [1] RSPM (R 4.5.0)
##  usethis              * 3.2.1     2025-09-06 [2] RSPM (R 4.5.0)
##  vctrs                  0.7.1     2026-01-23 [1] RSPM (R 4.5.0)
##  vipor                  0.4.7     2023-12-18 [1] RSPM (R 4.5.0)
##  viridis                0.6.5     2024-01-29 [1] RSPM (R 4.5.0)
##  viridisLite            0.4.3     2026-02-04 [1] RSPM (R 4.5.0)
##  withr                  3.0.2     2024-10-28 [2] RSPM (R 4.5.0)
##  xfun                   0.56      2026-01-18 [2] RSPM (R 4.5.0)
##  XML                    3.99-0.22 2026-02-10 [1] RSPM (R 4.5.0)
##  xtable                 1.8-4     2019-04-21 [2] RSPM (R 4.5.0)
##  XVector                0.50.0    2025-10-29 [1] Bioconductor 3.22 (R 4.5.2)
##  yaml                   2.3.12    2025-12-10 [2] RSPM (R 4.5.0)
## 
##  [1] /__w/_temp/Library
##  [2] /usr/local/lib/R/site-library
##  [3] /usr/local/lib/R/library
##  * ‚îÄ‚îÄ Packages attached to the search path.
## 
## ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ</code></pre>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="index.html#cb7-1" tabindex="-1"></a><span class="fu">proc.time</span>()</span></code></pre></div>
<pre><code>##    user  system elapsed 
##  11.601   1.031  12.284</code></pre>
<p>This book was last updated at 2026-02-11 21:05:06.052654.</p>
</div>
<div id="license" class="section level2 unnumbered hasAnchor">
<h2>License<a href="index.html#license" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.</p>

</div>
</div>
<div class="container">
    <footer class="site-footer">

      <center>
<p class="powered-by">
¬© 2011-2026. All thoughts and opinions here are my own. The icon was designed by <a href="https://www.linkedin.com/in/mauricio-guzman-6529b551/">Mauricio Guzm√°n</a> and is inspired by <a href="https://en.wikipedia.org/wiki/Huichol">Huichol culture</a>; it represents my community building interests.
</p>

<p class="powered-by copyright-license-text">
This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">CC BY NC SA 4.0</a>
</p>

<p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">
      <img src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" alt="CC icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/by.svg" alt="CC by icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/nc.svg" alt="CC NC icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/sa.svg" alt="CC SA icon" width="32px">
    </a>
</p>

<div id="clustrmaps-widget"></div><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=300&t=m&d=Zo-deincaJGsqhqjIWsujtdoNOMJsnnZLefKmxDKVv8&co=124b73&cmo=dd53ff&cmn=4ec724'></script>

<p class="powered-by">
Published with <a href="https://bookdown.org/" target="_blank" rel="noopener">Bookdown</a>
</p>

</center>



<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-45299226-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-45299226-1');
</script>

</footer>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="introducci√≥n-a-r-y-positron.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
