---
title: "Intro RNA-seq LCG-UNAM 2025"
author: "Leonardo Collado-Torres"
site: bookdown::bookdown_site
apple-touch-sin: "icon_192.png"
apple-touch-icon-size: 192
favicon: "icon_32.png"
github-repo: "lcolladotor/rnaseq_LCG-UNAM_2025"
documentclass: book
output:
  bookdown::gitbook:
    includes:
      after_body: map_and_ga.html
    css: "style.css"
    
---

# Overview {-}

Here you can find the files for the January 2025 introduction to R, RStudio and RNA-sequencing (RNA-seq) course for [LCG-UNAM](https://lcg.unam.mx/) at [CCG-UNAM](https://ccg.unam.mx/) (January 28-31, 2025). The rest of the chapters will be in Spanish.

Instructor:

* [**Leonardo Collado-Torres**](http://lcolladotor.github.io/): [team documentation website](https://lcolladotor.github.io/bioc_team_ds/), [Bluesky](https://bsky.app/profile/lcolladotor.bsky.social)

Teach assistants:

* Danna Ocampo Luna [Bluesky](https://bsky.app/profile/genomic-swallow.bsky.social)

Guest presenters:

* [Renee García-Flores](https://reneegf.github.io/), [Bluesky](https://bsky.app/profile/freneegf.bsky.social)

<!--
* [Daianna González Padilla](https://daianna21.github.io/), [Bluesky](https://bsky.app/profile/daiannaglez.bsky.social)
* [Melissa Mayén Quiroz](https://melii99.github.io/), [Bluesky](https://bsky.app/profile/meli-mq99.bsky.social)
-->

Course Coordinators:

* Heladia Salgado
* Julio Collado Vides

Download the materials for this course with `usethis::use_course('lcolladotor/rnaseq_LCG-UNAM_2025')` or view online at [**lcolladotor.github.io/rnaseq_LCG-UNAM_2025**](http://lcolladotor.github.io/rnaseq_LCG-UNAM_2025).

## Code of Conduct {-}

We'll follow the CDSB code of conduct [comunidadbioinfo.github.io/codigo-de-conducta/](https://comunidadbioinfo.github.io/codigo-de-conducta/) as well as version 1.3.0 of the Bioconductor code of conduct [bioconductor.github.io/bioc_coc_multilingual/](https://bioconductor.github.io/bioc_coc_multilingual/). 

For reporting any violations of the code of conduct, report them to the Instructor and/or Course Coordinators.

## Course Schedule {-}

_Local times in Cuernavaca, Mexico_

* Tuesday January 28:
  - 9-11 am: introduction to R, RStudio, `r BiocStyle::CRANpkg("usethis")` and `r BiocStyle::CRANpkg("here")`
  - 12-2 pm: introduction to GitHub Copilot with RStudio, RStudio projects, and connecting RStudio projects to GitHub
* Wednesday January 29:
  - 9-11 am: introduction to Bioconductor
  - 12-2 pm: expression data R/Bioconductor objects and visualization: `r BiocStyle::Biocpkg("SummarizedExperiment")` and `r BiocStyle::Biocpkg("iSEE")`
* Thursday January 30:
  - 9-11 am: downloading RNA-seq data using `r BiocStyle::Biocpkg("recount3")`
  - 12-2 pm: making your own website with `r BiocStyle::CRANpkg("postcards")` by [Renee García-Flores](https://reneegf.github.io/)
* Friday January 31:
  - 9-11 am: exploring statistical models with `r BiocStyle::Biocpkg("ExploreModelMatrix")`
  - 12-2 pm: differential expression analysis with `r BiocStyle::Biocpkg("limma")`
    - + review
    - + _R/Bioconductor-powered Team Data Science_ overview + open discussion (career paths, networking, etc)
    - [2025 LIBD summer internship program](https://www.libd.org/summer-internship): application closes ?? (previously early March).
  
## External links {-}

* [Cursos LCG](https://cursos.lcg.unam.mx/course/view.php?id=235)
* [GitHub source code](https://github.com/lcolladotor/rnaseq_LCG-UNAM_2025)
* Zoom (ask)
* [LCG-UNAM RStudio server](http://132.248.220.108:8787/)
* [CDSB](https://comunidadbioinfo.github.io/). 
* [Slack CDSB Mexico](https://comunidadbioinfo.slack.com), in particular the [`intro_rnaseq_lcg_2025` channel](https://comunidadbioinfo.slack.com/archives/C089V30HHP0). You can join this Slack workspace through [this link](https://join.slack.com/t/comunidadbioinfo/shared_invite/zt-8lsvpm84-Fne1W0hadk6cpjgJS17Tnw).
* [LieberInstitute/template_project](https://github.com/LieberInstitute/template_project)
* Example real project by [Daianna González Padilla](https://daianna21.github.io/): [LieberInstitute/smoking-nicotine-mouse](https://github.com/LieberInstitute/smoking-nicotine-mouse)
  

## Course Prerequisites {-}

Install R 4.4.x from [CRAN](https://cran.r-project.org/) then install the following R packages:

```{r install, eval = FALSE}
## For installing Bioconductor packages
if (!requireNamespace("BiocManager", quietly = TRUE)) {
    install.packages("BiocManager")
}

## Install required packages
BiocManager::install(
    c(
        "usethis", ## Utilities
        "here",
        "biocthis",
        "lobstr",
        "postcards",
        "sessioninfo",
        "SummarizedExperiment", ## Main containers / vis
        "iSEE",
        "edgeR", ## RNA-seq
        "ExploreModelMatrix",
        "limma",
        "recount3",
        "pheatmap", ## Visualization
        "ggplot2",
        "patchwork",
        "RColorBrewer",
        "ComplexHeatmap",
        "spatialLIBD" ## Advanced
    )
)
```

You will also need to install [RStudio](https://rstudio.com/products/rstudio/download/#download) version RStudio 2024.12.0 or newer.

## Course Evaluation {-}

* Participation
  - (main evidence) written and/or verbal participation during class
    - This is tracked by day by the TA. Each day counts as a category.
  - (optional, strongly encouraged for your own career) Community building: CDSB Slack (introduce yourself), Bluesky, GitHub, etc. These are tracked by the TA: please let the TA know of your participations.
    - GitHub issues on the course repository
    - CDSB Slack [#bienvenida](https://comunidadbioinfo.slack.com/archives/CAB6CHXUY) channel
    - Bioconductor Slack [#introductions](https://community-bioc.slack.com/archives/C36B0HK7W) channel
    - Bluesky posts related to the course
  - Grading:
    - 0%: no participation
    - 25%: participation in only 1 out of 8 categories (4 days, 4 public participation categories)
    - 50%: participation in 2 out of 8 categories 
    - 75%: participation in 3 out of 8 categories 
    - 100%: participation in 4 or more categories
* Public work
  - (main evidence) Taking notes on your GitHub class notes repositories
  - (optional) Deploy an iSEE `shiny` app on shinyapps.io
  - (advanced and optional) R package with notes following `usethis::create_package()` + `biocthis::use_bioc_pkg_templates()`.
* Final project
  - (main evidence) code in a public GitHub repository (different from the notes one) following the structure from [LieberInstitute/template_project](https://github.com/LieberInstitute/template_project)
  - (optional) a rendered Rmd file (RPubs, GitHub, etc) on a public URL
  - (optional advanced) Copy this repository, remove all the Rmds except the index.Rmd and edit accordingly. GitHub Actions will then help you render it. You'll need to fetch the `gh-pages` branch and at least make one commit for the HTML to be available such as this [commit](https://github.com/lcolladotor/rnaseq_LCG-UNAM_2021/commit/e15bd6c234dd0ca4233050a4a0a42184994fef3b). You might also run into [this GitHub default permission setting](https://github.com/ad-m/github-push-action/issues/96#issuecomment-1396347833) that you'll need to change.
  
The final grade will be a weighted average from the evaluation by the TA (20%), your public work (40%) and your final project (40%).

Projects, public work and notes are due at 9 am US Eastern on Monday February 10th, though we recommend doing any work related to this course by Friday February 7th.

## R session information {-}

Details on the R version used for making this book. The source code is available at [`lcolladotor/rnaseq_LCG-UNAM_2025`](https://github.com/lcolladotor/rnaseq_LCG-UNAM_2025).

```{r session_packages, eval = TRUE, message = FALSE}
## Load the package at the top of your script
library("sessioninfo")

##  Utilities
library("BiocStyle")
library("biocthis")
library("here")
library("lobstr")
library("postcards")
library("usethis")
library("sessioninfo")

## Main containers / vis
library("SummarizedExperiment")
library("iSEE")

## RNA-seq
library("edgeR")
library("ExploreModelMatrix")
library("limma")
library("recount3")

## Visualization
library("ComplexHeatmap")
library("ggplot2")
library("patchwork")
library("pheatmap")
library("RColorBrewer")

## Advanced
library("spatialLIBD")
```

```{r session_info}
## Reproducibility information
options(width = 120)
session_info()
proc.time()
```

This book was last updated at `r Sys.time()`.

## License  {-}

<a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.

