---
title: "Intro RNA-seq LCG-UNAM 2026"
author: "Leonardo Collado-Torres"
site: bookdown::bookdown_site
apple-touch-sin: "icon_192.png"
apple-touch-icon-size: 192
favicon: "icon_32.png"
github-repo: "lcolladotor/rnaseq_LCG-UNAM_2026"
documentclass: book
output:
  bookdown::gitbook:
    includes:
      after_body: map_and_ga.html
    css: "style.css"
    
---

# Overview {-}

Here you can find the files for the February 2026 introduction to R, RStudio and RNA-sequencing (RNA-seq) course for [LCG-UNAM](https://lcg.unam.mx/) at [CCG-UNAM](https://ccg.unam.mx/) (February 10-13, 2026). The rest of the chapters will be in Spanish.

Instructor:

* [**Leonardo Collado-Torres**](http://lcolladotor.github.io/): [team documentation website](https://lcolladotor.github.io/bioc_team_ds/), [Bluesky](https://bsky.app/profile/lcolladotor.bsky.social)

Teaching assistants:

* Maria Jose Palma, [Bluesky](), [LinkedIn]()

Guest presenters:

* [Melissa May√©n Quiroz](https://melii99.github.io/), [Bluesky](https://bsky.app/profile/meli-mq99.bsky.social)
* [Maria Gutierrez-Arcelus](https://www.gutierrezarceluslab.com/), [Bluesky](https://bsky.app/profile/mariagtac.bsky.social)
* [Gabriel Ram√≠rez-Vilchis](https://gabrielramirezv.github.io/), [Bluesky](https://bsky.app/profile/gabrielramirezv.bsky.social)
* [Daianna Gonz√°lez Padilla](https://daianna-glez.github.io/), [Bluesky](https://bsky.app/profile/daiannaglez.bsky.social)


<!--
* [Renee Garc√≠a-Flores](https://reneegf.github.io/), [Bluesky](https://bsky.app/profile/freneegf.bsky.social)
-->

Course Coordinators:

* Heladia Salgado
* Julio Collado Vides

Download the materials for this course with `usethis::use_course('lcolladotor/rnaseq_LCG-UNAM_2026')` or view online at [**lcolladotor.github.io/rnaseq_LCG-UNAM_2026**](http://lcolladotor.github.io/rnaseq_LCG-UNAM_2026).

## Code of Conduct {-}

We'll follow the CDSB code of conduct [comunidadbioinfo.github.io/codigo-de-conducta/](https://comunidadbioinfo.github.io/codigo-de-conducta/) as well as version 1.4.0 of the Bioconductor code of conduct [bioconductor.github.io/bioc_coc_multilingual/](https://bioconductor.github.io/bioc_coc_multilingual/). 

For reporting any violations of the code of conduct, report them to the Instructor and/or Course Coordinators.

## Course Schedule {-}

_Local times in Cuernavaca, Mexico_

* Tuesday February 10:
  - 9-11 am: introduction to R, Positron, `r BiocStyle::CRANpkg("usethis")` and `r BiocStyle::CRANpkg("here")`
  - 12-2 pm: introduction to GitHub Copilot with Positron, and connecting Positron to GitHub
* Wednesday February 11:
  - 9-11 am: introduction to Bioconductor
  - 12-2 pm: expression data R/Bioconductor objects and visualization: `r BiocStyle::Biocpkg("SummarizedExperiment")` and `r BiocStyle::Biocpkg("iSEE")`
    - 12-1 pm: `r BiocStyle::Biocpkg("SummarizedExperiment")` intro
    - 1-2 pm: [Melissa May√©n Quiroz](https://melii99.github.io/) will teach you about `r BiocStyle::Biocpkg("iSEE")` and showcase an `r BiocStyle::Biocpkg("iSEE")` app she made for [LieberInstitute/Hb_VTA_projector_profiling](https://github.com/LieberInstitute/Hb_VTA_projector_profiling).
* Thursday February 12:
  - 9-11 am: making your own website with `r BiocStyle::CRANpkg("postcards")` 
  - 12-2 pm: downloading RNA-seq data using `r BiocStyle::Biocpkg("recount3")` by [Gabriel Ram√≠rez-Vilchis](https://gabrielramirezv.github.io/)
* Friday February 13:
  - 9-11 am: exploring statistical models with `r BiocStyle::Biocpkg("ExploreModelMatrix")`
  - 12-1:30 pm: current examples of RNA-seq data analysis
    - Guest presentation by [Maria Gutierrez-Arcelus](https://www.gutierrezarceluslab.com/)
    - Guest presentation by [Gabriel Ram√≠rez-Vilchis](https://gabrielramirezv.github.io/)
    - Guest presentation by [Daianna Gonz√°lez Padilla](https://daianna-glez.github.io/)
  - 1:30-2 pm: differential expression analysis with `r BiocStyle::Biocpkg("limma")`
    - + review
    - + _R/Bioconductor-powered Team Data Science_ overview + open discussion (career paths, networking, etc)
  
## External links {-}

* [Cursos LCG](https://cursos.lcg.unam.mx/course/view.php?id=269#section-2)
* [GitHub source code](https://github.com/lcolladotor/rnaseq_LCG-UNAM_2026)
* Zoom (ask)
* [LCG-UNAM RStudio server](http://132.248.220.108:8787/)
* [CDSB](https://comunidadbioinfo.github.io/) 
* [Slack CDSB Mexico](https://comunidadbioinfo.slack.com), in particular the [`intro_rnaseq_lcg_2026` channel](https://comunidadbioinfo.slack.com/archives/C0AC8U6UPD2).
  - You can join this Slack workspace through [this link](https://join.slack.com/t/comunidadbioinfo/shared_invite/zt-8lsvpm84-Fne1W0hadk6cpjgJS17Tnw).
* [Bioconductor Zulip](https://community-bioc.zulipchat.com/) community
* [LieberInstitute/template_project](https://github.com/LieberInstitute/template_project)
* Example real project by [Daianna Gonz√°lez Padilla](https://daianna-glez.github.io/): [LieberInstitute/smoking-nicotine-mouse](https://github.com/LieberInstitute/smoking-nicotine-mouse)
  

## Course Prerequisites {-}

* Install R 4.5.x from [CRAN](https://cran.r-project.org/).
* Install [Positron](https://positron.posit.co/download.html) which will be our Interactive Development Environment (IDE) of choice for the course.
  - Alternatively, you could install [RStudio IDE](https://rstudio.com/products/rstudio/download/#download) version RStudio 2026.01.0 or newer.
* In either case, you will also need to install [Air](https://posit-dev.github.io/air/) for automatic code formatting.
* You will also have to requesst access to the [GitHub Education](https://github.com/education) pack in order to use GitHub Copilot Pro ü§ñ.
* Finally install the following R packages:

```{r install, eval = FALSE}
## For installing Bioconductor packages
if (!requireNamespace("BiocManager", quietly = TRUE)) {
    install.packages("BiocManager")
}

## Install required packages
BiocManager::install(
    c(
        "usethis", ## Utilities
        "here",
        "biocthis",
        "lobstr",
        "postcards",
        "sessioninfo",
        "SummarizedExperiment", ## Main containers / vis
        "iSEE",
        "edgeR", ## RNA-seq
        "ExploreModelMatrix",
        "limma",
        "recount3",
        "pheatmap", ## Visualization
        "ggplot2",
        "patchwork",
        "RColorBrewer",
        "ComplexHeatmap",
        "spatialLIBD" ## Advanced
    )
)
```

### Getting set up {-}

Here are more step by step instructions for the course prerequisites.

You must install [R](https://cran.r-project.org) and [Positron](https://positron.posit.co/download.html) (available from [Posit](https://posit.co/)) on your computing environment. As an alternative to Positron, you can install [RStudio IDE](https://posit.co/products/open-source/rstudio/?sid=1).

These are three **different** applications that must be installed separately before they can be used together:

-   `R` is the core underlying programming language and computing engine that we will be learning in this course

-   `RStudio IDE` is an interface into R that makes many aspects of using and programming R simpler

-   `Positron` is a new interface for working with R developed by [Posit](https://posit.co/) (formerly RStudio). All new features are being implemented in `Positron`, but as it was just recently released, it might have some bugs. If you are choosing between `RStudio IDE` and `Positron`, check <https://positron.posit.co/faqs.html#is-rstudio-going-away>.

`R`, `RStudio IDE`, and `Positron` are available for Windows, macOS, and most flavors of Unix and Linux. Please download the version that is suitable for your computing setup.

Throughout the course, we will make use of numerous R add-on packages that must be installed over the Internet. Packages can be installed using the `install.packages()` function in R. For example, to install the `tidyverse` package, you can run

```{r}
#| eval: false
install.packages("tidyverse")
```

in the R console.

We will also use [Git](https://git-scm.com/) to version control your code, which you can install following the [*Happy Git and GitHub for the useR*](https://happygitwithr.com/) book. Through the [GitHub Education](https://github.com/education) pack we will get free access to [GittHub Copilot Pro](https://github.com/features/copilot) to help us write code: think about it as an enhanced auto-complete. Finally, we'll use [Air](https://posit-dev.github.io/air/) for automatically formatting our R code.

### How to Download R for Windows {-}

Go to <https://cran.r-project.org> and

1.  Click the link to "Download R for Windows"

2.  Click on "base"

3.  Click on "Download R 4.5.2 for Windows"

![Video Demo for Downloading R for Windows](https://lcolladotor.github.io/jhustatcomputing/videos/downloadRWindows.gif){width="80%"}

_The version in the video is not the latest version of R. Please download the latest version._

### How to Download R for the Mac {-}

Go to <https://cran.r-project.org> and

1.  Click the link to "Download R for (Mac) OS X".

2.  Click on "R-4.5.2-arm64.pkg" (for Apple Silicon Macs)

![Video Demo for Downloading R for the Mac](https://lcolladotor.github.io/jhustatcomputing/videos/downloadRMac.gif){width="80%"}

_The version in the video is not the latest version of R. Please download the latest version._

### How to Download `RStudio IDE` {-}

Go to <https://posit.co/> (formerly [https://rstudio.com]()) and

1.  Click on "Free & Open Source" in the top menu

2.  Click on "RStudio IDE" in the drop down menu

3.  Click the button that says "DOWNLOAD RSTUDIO DESKTOP"

4.  Click the button under "Download RStudio" which will take you to <https://posit.co/download/rstudio-desktop/>

5.  Under the section "All Installers and Tarballs" choose the file that is appropriate for your operating system.

![Video Demo for Downloading RStudio](https://lcolladotor.github.io/jhustatcomputing/videos/downloadRStudio.gif){width="80%"}

_NOTE: The video shows how to download RStudio for the Mac but you should download RStudio for whatever computing setup you have._

### How to Download Positron {-}

Go to <https://posit.co/> and

1.  Click on "Free & Open Source" in the top menu

2.  Click on "Positron" in the drop down menu

3.  Click on "Download Positron" which will take you to <https://positron.posit.co/download.html>

4.  Read the license agreement.

5.  Accept the license agreement.

6.  On the table with all the download options, choose the file that is appropriate for your operating system.

### How to Download Git {-}

Install `Git` on your computer following the detailed instructions at <https://happygitwithr.com/install-git>, which will depend on your operating system.

### Create your GitHub account {-}

Create a personal GitHub account following the instructions at <https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account>.

Following [Jeff Leek](https://bsky.app/profile/jtleek.bsky.social)'s advice on [**How to be a modern scientist**](https://leanpub.com/modernscientist) that was part of my team's bootcamps at <https://lcolladotor.github.io/bioc_team_ds/how-to-be-a-modern-scientist.html>, try to choose a username that will be the same one you use on your email and other work-related social media platforms such as Bluesky, LinkedIn, etc.

<iframe width="560" height="315" src="https://www.youtube.com/embed/93hA-iPWsbs?si=o-Ekbw7iOSoa5j6e" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>

If you are using `Positron`, under the "Accounts" options (bottom left corner), click on "Sign in with GitHub \[...\]", provide the 8 digit code, and authorize "posit-dev".

### How to Apply for GitHub Education and configure GitHub Copilot {-}

Go to <https://github.com/education> and

1.  Click on "Join GitHub education" (likely <https://github.com/settings/education/benefits?locale=en-US>)
2.  Sign in to your GitHub account if you haven't logged in
3.  Click on "Start Application"
4.  There will be a few options to verify that you are a student. One of them is by having a photo of your student ID with dates showing that your student ID is still valid. I've used a screenshot of my Faculty website at <https://publichealth.jhu.edu/faculty/4647/leonardo-collado-torres> when validating my information.
5.  Wait a few **days** for GitHub to process your application.
6.  Once your application has been approved, if you are using `RStudio IDE` follow the instructions at <https://docs.posit.co/ide/user/ide/guide/tools/copilot.html> to enable GitHub Copilot in RStudio IDE. I have the "Index project files with GitHub Copilot" checkbox enabled on my preferences.
7.  If you are using `Positron`, follow the instructions at <https://positron.posit.co/assistant> to enable "Positron assistant". Then under the "Positron Assistant" chat, click on "Add Anthropic as a Chat Provider", then select "GitHub Copilot" and click "Sign in".

### How to get Air (auto-code formatter) {-}

Go to <https://posit-dev.github.io/air/> and

1.  Click on "RStudio" if you are using `RStudio IDE`. That will take you to <https://posit-dev.github.io/air/editor-rstudio.html> where you will find the detailed installation instructions.

    -   Note that I recommend enabling the `Tools -> Global Options -> Code -> Saving` and check `Reformat documents on save` option. Particularly if you are working on your own code.

2.  Click on "VS Code and Positron" if you are using `Positron`. That will take you to <https://posit-dev.github.io/air/editor-vscode.html> where some options and configuration steps are described. Note however, that `Air` is already available from `Positron`. You will need to open your `settings.json` file from the Command Palette (`Shift + Cmd + P` on Mac/Linux, `Shift + Ctrl + P` on Windows):

    -   Run `Preferences: Open Workspace Settings (JSON)` to modify workspace specific settings (recommended).

    -   Run `Preferences: Open User Settings (JSON)` to modify global user settings (preferred option if you are not working with code from others). Then edit your `settings.json` with the relevant lines about R code formatting and `Air` as shown below:

        ```{json}
        {
            "files.associations": {
                "renv.lock": "json"
            },
            "positron.assistant.gitIntegration.enable": true,
            "positron.assistant.showTokenUsage.enable": true,
            "positron.assistant.toolDetails.enable": true,
            "[r]": {
                "editor.formatOnSave": true,
                "editor.defaultFormatter": "Posit.air-vscode"
            }
        }
        ```

    1.  You can also configure a few options at [positron://settings/air.dependencyLogLevels](positron://settings/air.dependencyLogLevels) and related settings.

## Course Evaluation {-}

* Participation
  - (main evidence) written and/or verbal participation during class
    - This is tracked by day by the TA. Each day counts as a category.
  - (optional, strongly encouraged for your own career) Community building: CDSB Slack (introduce yourself), Bluesky, GitHub, etc. These are tracked by the TA: please let the TA know of your participations.
    - GitHub issues on the course repository
    - CDSB Slack [#bienvenida](https://comunidadbioinfo.slack.com/archives/CAB6CHXUY) channel
    - Bioconductor Zulip [#introductions](https://community-bioc.zulipchat.com/#narrow/channel/507510-introductions) channel
    - Bluesky posts related to the course
  - Grading:
    - 0%: no participation
    - 25%: participation in only 1 out of 8 categories (4 days, 4 public participation categories)
    - 50%: participation in 2 out of 8 categories 
    - 75%: participation in 3 out of 8 categories 
    - 100%: participation in 4 or more categories
* Public work
  - (main evidence) Taking notes on your GitHub class notes repositories
  - (optional) Deploy an iSEE `shiny` app on shinyapps.io
  - (advanced and optional) R package with notes following `usethis::create_package()` + `biocthis::use_bioc_pkg_templates()`.
* Final project
  - (main evidence) code in a public GitHub repository (different from the notes one) following the structure from [LieberInstitute/template_project](https://github.com/LieberInstitute/template_project)
  - (optional) a rendered Rmd file (RPubs, GitHub, etc) on a public URL
  - (optional advanced) Copy this repository, remove all the Rmds except the index.Rmd and edit accordingly. GitHub Actions will then help you render it. You'll need to fetch the `gh-pages` branch and at least make one commit for the HTML to be available such as this [commit](https://github.com/lcolladotor/rnaseq_LCG-UNAM_2021/commit/e15bd6c234dd0ca4233050a4a0a42184994fef3b). You might also run into [this GitHub default permission setting](https://github.com/ad-m/github-push-action/issues/96#issuecomment-1396347833) that you'll need to change.
  
The final grade will be a weighted average from the evaluation by the TA (20%), your public work (40%) and your final project (40%).

Projects, public work and notes are due at 9 am US Eastern on Monday February 23rd, though we recommend doing any work related to this course by Friday February 20th.

## R session information {-}

Details on the R version used for making this book. The source code is available at [`lcolladotor/rnaseq_LCG-UNAM_2026`](https://github.com/lcolladotor/rnaseq_LCG-UNAM_2026).

```{r session_packages, eval = TRUE, message = FALSE}
## Load the package at the top of your script
library("sessioninfo")

##  Utilities
library("BiocStyle")
library("biocthis")
library("here")
library("lobstr")
library("postcards")
library("usethis")
library("sessioninfo")

## Main containers / vis
library("SummarizedExperiment")
library("iSEE")

## RNA-seq
library("edgeR")
library("ExploreModelMatrix")
library("limma")
library("recount3")

## Visualization
library("ComplexHeatmap")
library("ggplot2")
library("patchwork")
library("pheatmap")
library("RColorBrewer")

## Advanced
library("spatialLIBD")
```

```{r session_info}
## Reproducibility information
options(width = 120)
session_info()
proc.time()
```

This book was last updated at `r Sys.time()`.

## License  {-}

<a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.

